package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.26.7",
   "description": "Dead simple Object schema validation",
   "main": "lib/index.js",
+  "module": "src/index.js",
```

Comment:
this isn't right, `module` should point to an esm build not source code. The rest of the code here may not be something that can be run in a browser depending on which proposals and bespoke babel plugins i'm running. Practically I do think the code is pretty standard es6 but this isn't safe generally. In general i'd like to handle this via a flat rollup build

Reply:
Makes sense. Thank you for the feedback and clear direction. I'll add an esm build w/ rollup and resubmit.

Reply:
K. please see my latest updates. I saw you already have a rollup config setup, just not building. I added a `build:esm` script to run rollup, called it from the `build` script and added `/dist/yup.esm.js` to pkg.files. Locally I ran this in a project with `yarn link` and seems to grab the module and work.  Let me know if you have something else in mind or if I'm missing something.

Reply:
ya i had started the work to add a rollup config but didn't finish. Honestly i'm not sure if it even works, may need to test it

Reply:
@jquense One idea I had was running tests on the es bundle too with a secondary jest config. I reading that react did that and found the article at https://reactjs.org/blog/2017/12/15/improving-the-repository-infrastructure.html#running-tests-on-compiled-bundles Let me know if that is a direction you'd like to go to merge this or if you have something else in mind.